templates/named.conf.options.erb

```diff
@@ -1,5 +1,5 @@
 options {
-	directory "/var/cache/bind";
```

Comment:
This seems fine, but before this being /var/cache/bind, it will change to /etc/bind/zones, is that right? (for debian)

On redhat this will change to "/var/named/" Can you also confirm this is appropriate?


I think that this directory is for the caching resolver right? If so then maybe we should just leave it as /var/cache/bind and have puppet manage it (create it)